lib/shared/units.js

```diff
@@ -29,7 +29,7 @@ const prettyBytes = require('pretty-bytes')
  * @description
  * 1 MB = 1e+6 B
```

Comment:
Update the description too?

Reply:
Does #1669 help?

Reply:
Will do, I need to replace prettyBytes with something else too.Actually I reverted this as it is unrelated to the use of the sdk.

Reply:
Actually I reverted this as it is unrelated to the use of the sdk.